docs/_static/llama-stack-spec.yaml

```diff
@@ -2721,10 +2721,6 @@ components:
               description: >-
                 A URL of the image or data URL in the format of data:image/{type};base64,{data}.
                 Note that URL could have length limits.
-            data:
```

Comment:
let's keep OpenAPI spec the same and separate from this PR for adding benchmark.